superset-frontend/plugins/plugin-chart-table/src/controlPanel.tsx

```diff
@@ -108,6 +108,9 @@ const all_columns: typeof sharedControls.groupby = {
   optionRenderer: c => <ColumnOption showType column={c} />,
   valueRenderer: c => <ColumnOption column={c} />,
   valueKey: 'column_name',
+  shouldMapStateToProps() {
```

Comment:
Do we need it here and in other controls in Table? `mapStateToProps` has only 2 arguments, so we do not need to run it every time

Reply:
No, you're correct, treated the object as two params